lib/facter/iptables_persistent_version.rb

```diff
@@ -6,9 +6,9 @@
     # Throw away STDERR because dpkg >= 1.16.7 will make some noise if the
     # package isn't currently installed.
     cmd = "dpkg-query -Wf '${Version}' netfilter-persistent 2>/dev/null"
-    version = Facter::Core::Execution.execute(cmd)
+    version = Facter::Core::Execution.execute(cmd, { on_fail: false })
```

Comment:
might as well use `on_fail: nil` and avoid the dual typed variable (can be either string or boolean)?